src/applications/virtual-agent/components/webchat/WebChat.jsx

```diff
@@ -17,6 +18,18 @@ import {
 
 const renderMarkdown = text => MarkdownRenderer.render(text);
 
+export function cardActionMiddleware(next, card) {
+  // Track decision letter downloads
+  if (card.cardAction.value.includes('/v0/claim_letters/')) {
+    recordEvent({
+      event: 'file_download',
+      'button-click-label': 'Decision Letter',
```

Comment:
What is the significance of this label?

Reply:
This will record the event in GA as `Download - Decision Letter` and we can filter down to it by page so we’ll know that it was from our page.

Reply:
So normally the button label doesn't include the word `decision letters` so we set our own label